pkg/api/frontendsettings.go

```diff
@@ -167,6 +167,16 @@ func (hs *HTTPServer) getFrontendSettingsMap(c *models.ReqContext) (map[string]i
 		}
 	}
 
+	version := setting.BuildVersion
+	commit := setting.BuildCommit
+	buildstamp := setting.BuildStamp
+
+	if setting.AnonymousHideVersion && !c.IsSignedIn {
```

Comment:
Shouldn't we avoid rendering the version rather than overwriting it?

Reply:
@bergquist Could the idea here be for the version to be invisible to the client (including JavaScript)?

Reply:
doh! ofc.

Reply:
Good point, I did this because I was concerned that the frontend might expect a version to be formatted like a version string, but I don't think it does. I'll send it empty from the backend and do something on the frontend corresponding that.